eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -381,15 +402,16 @@ class Router(nodeParams: NodeParams, watcher: ActorRef, initialized: Option[Prom
       stay
 
     case Event('channels, d) =>
-      sender ! d.channels.values
+      sender ! d.channels.values.map(_.ann)
       stay
 
-    case Event('updates, d) =>
-      sender ! (d.updates ++ d.privateUpdates).values
+    case Event('channelsMap, d) =>
```

Comment:
nit: I think it's clearer if this one is called `'channels` and the previous one is renamed `'channelAnnouncements`

Reply:
I like the `channels`/`updates`/`nodes` triplet though.

Reply:
Ok, I guess it maps cleanly to the spec structure (channel_announcement, channel_update, node_announcement)